indexer/src/test/java/org/uniprot/store/indexer/uniprotkb/converter/UniProtKBEntryTaxonomyConverterTest.java

```diff
@@ -74,29 +74,15 @@ void convertModelOrganism() {
                         "Homo sapiens",
                         "Human",
                         "Homo sapian",
-                        "HUMAN",
+                        //                        "HUMAN",
```

Comment:
I think we will need the mnemonic for organism search. For taxonomy (lineage) we do not need.. (we can double check it with Sangya)

Reply:
Jie said we didn't need this one, and uniprot.org doesn't do it either.

for: https://www.ebi.ac.uk/proteins/api/taxonomy/id/55

https://www.uniprot.org/uniprot/?query=taxonomy%3A%22Polyangium+%5B55%5D%22&sort=score (can search by scientific name)
https://www.uniprot.org/uniprot/?query=9DELT&sort=score (mnemonic hits the second part of the entry ID)
https://www.uniprot.org/uniprot/?query=taxonomy%3A9delt&sort=score (mnemonic not indexed)

... waiting for a chat with sangya to check if this is fine to follow.

Reply:
Confirmed not needed by Sangya